shared/src/commonMain/kotlin/com/example/tasky/util/StringExtension.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.tasky.util
+
+fun String.getAvatarDisplayName(): String {
+    val splitName = this.trim().split(" ")
+    return when (splitName.size) {
+        0 -> ""
+        1 -> splitName[0].take(2)
+        else -> splitName[0][0].toString() + splitName.last()[0]
+    }
+}
```

Comment:
This would break for a name with leading, trailing or multiple spaces in between, e.g. for `Philipp Lackner `, since that would result in an empty splitted list where you then try to access a character of

Reply:
I have added trim() which can avoid errors caused by leading and trailing spaces